.github/workflows/windows-x64.yml

```diff
@@ -119,6 +119,12 @@ jobs:
         run: |
           choco install visualstudio2019community --package-parameters "--add Microsoft.VisualStudio.Component.VC.Redist.MSM"
 
+      - name: DPI awareness
+        shell: powershell
+        run: |
+          $env:PATH += ";C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64"
+          mt.exe -manifest .\yubioath-desktop.exe.manifest -outputresource:.\release\yubioath-desktop.exe;#1
```

Comment:
Should the `#1` in the end be there?

Reply:
Yes, however, I don't think it's required.
https://docs.microsoft.com/en-us/windows/win32/sbscs/mt-exe

Reply:
Removed that part. Seems to work!